### PR TITLE
ASM-6183 Always disable ErrPrompt

### PR DESCRIPTION
**Pull request by asaenz**

### CODE REVIEW

File: lib/puppet/provider/importtemplatexml.rb [hunk 1]
```diff
@@ -153,48 +153,51 @@ def get_config_changes
...
+          changes["remove"]["attributes"]["BIOS.Setup.1-1"] << key
         else
-          changes['partial']['BIOS.Setup.1-1'][key] = @bios_settings[key]
+          changes["partial"]["BIOS.Setup.1-1"][key] = @bios_settings[key]
         end
       end
     end
+
+    # Always disable ErrPrompt so we can avoid a hangup due to a user needing to put input on the server manually
+    changes["partial"]["BIOS.Setup.1-1"]["ErrPrompt"] = "Disabled"
```
Comment (gavin-scott) [inline lib/puppet/provider/importtemplatexml.rb]: Is this the only real change? Everything else is just `'` changed to `"`?

**Comment by gavin-scott** [comment]
I think this is supposed to go into 8.2.0 -- can you reissue this against release/dell-asm-8.2.0?

Other than that :+1:

File: lib/puppet/provider/importtemplatexml.rb (hunk 1 above)
Comment (asaenz) [inline lib/puppet/provider/importtemplatexml.rb]: Yeah, this line is the only substantial change.

**Comment by asaenz** [comment]
Oh, whoops. I'll fix it. Though I'll leave this PR since we probably want it for both 8.2 and 8.3, yeah?

**Comment by asaenz** [comment]
8.2 PR: https://github.com/dell-asm/dell-idrac/pull/191

**Comment by gavin-scott** [comment]
The plan is to try to merge from 8.2 branch back into master periodically so that we have to do less manual cherry-picks. So should be able to do just 8.2 branch for 8.2 fixes and just master for 8.3 changes. My understanding is that it won't hurt to go ahead and merge this into master though, so I'll do that.

**Pull request merged**